js/client/modules/@arangodb/testsuites/endpoints.js

```diff
@@ -92,7 +96,7 @@ function endpoints (options) {
           };
         }
 
-        let result = tu.runInArangosh(options, instanceInfo, 'js/client/tests/endpoint-spec.js');
+        let result = tu.runInArangosh(options, instanceInfo, testPaths.endpoints[0]);
```

Comment:
loop or fixed path?

Reply:
here we directly match a file, tu.runInArangosh() expects one test file.